app/lib/utils/getFilteredOutTermsQuery.js

```diff
@@ -0,0 +1,20 @@
+const constants = require('../constants');
+
+function getFilter(searchType) {
+  if (searchType === constants.searchTypes.sexperts) {
+    const mustNotClause = [
+      { match: { name: 'Marie Stopes' } },
```

Comment:
Are these names exact matches? And if so, is that going to catch all instances of them?

Reply:
Partial matches and it will catch all instances.